docs/docs/launch-manual/sections/core-hacking/sections/building-pulsar.md

```diff
@@ -106,7 +107,7 @@ yarn build:apm
 Start Pulsar!
 
 ```sh
-yarn start
+yarn start --no-sandbox
```

Comment:
PR 276 (https://github.com/pulsar-edit/pulsar/pull/276) has been merged now so `--no-sandbox` can be removed entirely from here.

Reply:
I got rid of it so it is done